.github/workflows/nuget-publish.yml

```diff
@@ -0,0 +1,70 @@
+name: NuGet Publish
+
+on:
+  push:
+    branches: [ master ]
+
+env:
+  NUGET_DIR: ${{ github.workspace }}/nuget
+  NUGET_ARTIFACT: nuget
+
+jobs:
+  testing:
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    uses: ./.github/workflows/test.yml
+
+  pack: 
+    if: contains(github.event.head_commit.message, 'chore(release)') 
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+
+      - run: sed -E 's:RepositoryCommit>.+<\/:RepositoryCommit>${{ github.sha }}<\/:g' ButterCMS/ButterCMS.csproj > tmp && mv tmp ButterCMS/ButterCMS.csproj
+
+      - run: dotnet pack --configuration Release --output ${{ env.NUGET_DIR }}
```

Comment:
shellcheck:
```

In a.sh line 1:
dotnet pack --configuration Release --output ${{ env.NUGET_DIR }}
                                             ^-----------------^ SC2086 (info): Double quote to prevent globbing and word splitting.
                                               ^--------------^ SC2296 (error): Parameter expansions can't start with {. Double check syntax.
                                                                ^-- SC1083 (warning): This } is literal. Check expression (missing ;/\n?) or quote it.

Did you mean: 
dotnet pack --configuration Release --output "${{ env.NUGET_DIR }"}

For more information:
  https://www.shellcheck.net/wiki/SC2296 -- Parameter expansions can't start ...
  https://www.shellcheck.net/wiki/SC1083 -- This } is literal. Check expressi...
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...
```

Reply:
Resolved in https://github.com/ButterCMS/buttercms-csharp/pull/40/commits/f3d2d5e3f00c7bfcdd3405cd42a7cfb56b556a32